.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: python
+python:
+  - '3.6'
+
+install:
+  - pip install --upgrade sphinx
+
+script:
+  - ./scripts/update_docs.sh
+  - touch docs/.nojekyll
+
+branches:
+  only:
+    - master
+
+deploy:
+  provider: pages
+  local_dir: docs
+  skip_cleanup: true
+  email: yanhuil@google.com # To satisfy the CLA check, replace this with bot email.
+  github_token: $GITHUB_TOKEN # Set in travis-ci.org dashboard
```

Comment:
Currently, we're using a personal token with the `public_repo` scope. @rakyll suggested that we may create a bot user for the organization in the future and replace this token with one for the bot.

Reply:
Thanks for suggestion!